README.rst

```diff
@@ -563,6 +563,22 @@ their hardware ID instead of their name.
 
 If you still want to reference devices by their name, set this to ``False``.
 
+``OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    |  ``str``    |
++--------------+-------------+
+| **default**: | ``Device``  |
++--------------+-------------+
+
+When enabled, makes device verbose name configurable in the
+``Network Configuration administration``.
+
+You can use more specialized word rather than generic device.
+
```

Comment:
Please remove this extra newlines

Reply:
I followed the format of the readme that's why i have added new lines.

Reply:
You are right @codesankalp but we need to know where to use this newlines. Generally, newlines are used to separate paragraphs or new ideas and not sentences. Let's keep the documentation simple. we could write something like:

```
Defines the verbose_name attribute of Device model, which is displayed on admin site.
For example, if we want to change the verbose name of Device model at admin, we could write;

.. code-block: python

    OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME = 'Router'
```